### PR TITLE
Bug 1876368 - Ship v128.0.0 of the WebCompat System Addon

### PR DESCRIPTION
**Pull request by ksy36**
Edit: Converting this to 128.0.0 release and not just one patch

As publitas deployed a fix, we can remove the override :)

See https://bugzilla.mozilla.org/show_bug.cgi?id=1898532

I've checked a few catalogues and they work as expected.

### CODE REVIEW

**Comment by ksy36** [comment]
I'll add the rest of the patches for 128 to this PR, so converting it to a draft for now

File: src/injections/js/bug1896383-error-capturestacktrace-shim.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use strict";
+
+/*
+ * Bug 1896383 - Shim "captureStackTrace" in Error
+ * Webcompat issue #129144 - https://github.com/webcompat/web-bugs/issues/136865
```
Comment (wisniewskit) [inline src/injections/js/bug1896383-error-capturestacktrace-shim.js:9]: Silly nit, but 129144 -> 136865

File: src/injections/js/bug1897120-turnjs-zoom-fix.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use strict";
+
+/*
+ * Bug 1897120 - Override "MozTransform" in element.style and return false
+ * Webcompat issue #129144 - https://github.com/webcompat/web-bugs/issues/137038
```
Comment (wisniewskit) [inline src/injections/js/bug1897120-turnjs-zoom-fix.js:9]: Same silly nit

Comment (wisniewskit) [inline src/injections/js/bug1897120-turnjs-zoom-fix.js:9]: Le silly nit again

**Review by wisniewskit — APPROVED** [review]
Ship it real good.

**Comment by ksy36** [comment]
Thanks :)

**Pull request merged**